deploy/runtime/help/html/battery_life.htm

```diff
@@ -50,10 +51,18 @@ <h3 class="p_Heading3_atocs_" style="page-break-inside: avoid; page-break-after:
-<p class="p_Text"><span class="f_Text">For a more detailed description of the battery lifetime model, see Section 2.3 of DiOrio, N.; Dobos, A.; Janzou, S.; Nelson, A.; Lunstrom, B. (2015). Technoeconomic Modeling of Battery Energy Storage in SAM. 32 pp. NREL/TP-6A20-64641 (<a href="http://www.nrel.gov/docs/fy15osti/64641.pdf" class="weblink">PDF 2.6 MB</a>)</span></p>
-<p class="p_Note"><span class="f_CHNote">Note.</span><span class="f_Note"> To control the frequency of battery replacements as they degrade over time or to remove battery replacements from your analysis, expand the Battery Replacements panel to change the replacement options, and change the battery replacement costs shown on the <a href="oc_battery.htm" class="topiclink">Operating Costs</a> page as appropriate.</span></p>
+<p class="p_Text"><span class="f_Text">The battery life model determines battery's degradation, or reduction in available battery capacity over time. There are two degradation mechanisms:</span></p>
+<p class="p_List" style="text-indent: 0; padding-left: 24px; margin-left: 24px;"><span class="f_List" style="font-family: Arial,'Lucida Sans Unicode','Lucida Grande','Lucida Sans';display:inline-block;width:24px;margin-left:-24px">&#8226;</span><span class="f_List">Calendar degradation is a reduction in battery capacity due to age, regardless of how the battery is used.</span></p><p class="p_ListLast" style="text-indent: 0; padding-left: 24px; margin-left: 24px;"><span class="f_List" style="font-family: Arial,'Lucida Sans Unicode','Lucida Grande','Lucida Sans';display:inline-block;width:24px;margin-left:-24px">&#8226;</span><span class="f_ListLast">Cycle degradation is a reduction in battery capacity due to the number of charge/discharge cycles.</span></p><p class="p_Text"><span class="f_Text">SAM provides three options for modeling battery life:</span></p>
+<p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Cycle and calendar degradation</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">This option characterizes the battery life parameters using tables of cycle degradation and calendar degradation rates that you can customize. SAM applies the &nbsp;minimum of the the two types of degradation.</span></p>
+<p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Li-ion NMC/Graphite</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">This option for the Lithium-ion Nickel Manganese Cobalt (NMC) Oxide battery type uses an internal battery life model that you cannot modify. This model only considers cycle degradation.</span></p>
```

Comment:
I think the sentence about the NMC model only considering cycling degradation is not correct. In Kandler's paper, cells 5, 8, 10, and 11 are not cycled, and the model successfully predicts their calendar degradation. I think the second sentence about combining calendar and cycle degradation from the LMO/LTO model would be accurate for NMC, or we could get even more specific by talking about taking the minimum the lithium limited capacity and the negative electrode site limited capacity. The Li capacity includes both calendar and cycling terms, whereas Qneg only includes cycling.